lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
Hrm, I think this isn't guaranteed - the convention is that in the case of functions returning a non-nil error, you shouldn't rely on the values of *any* of the other return values. Neither Atoi nor ParseInt make the guarantee that the return will be 0 if err != nil, it *probably* will be since that's the default value but it's not specified AFAICT.